### DISCUSSION TITLE
How do I get (upgrade to) the latest live EMu client?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#352] [HOW TO...]
How do I get (upgrade to) the latest live EMu client?

**Comment by reguzik** [comment]
### DEPRECATED: 24 August 2026 post migration from v5.1 to v9

Read the following instructions and when you are finished run the EMu client install package:
- How to install/update the EMu **LIVE** client [googledoc](https://docs.google.com/document/d/1GJMkQcXWc_FaDOsGD4DqSua75ZanGkL3SkOU9ExlCcc/)
- How to install the EMu **TEST** client [googledoc](https://docs.google.com/document/d/1Ehc7ovcpCkuE-55sD-uCl65lURO-CPtbi-9bXae3TZM/)
 
To download the latest client install package navigate to this directory/folder:
- **LIVE**:  \\\\filer01\emu\clients\live  
- **TEST**: \\\\filer01\emu\clients\test 

IMPORTANT: 
- [Search cheat sheet](https://github.com/fieldmuseum/EMu-Documentation/files/14015681/EMu_Unicode_Cheatsheet_IE_20171019.pdf) - (from [Axiell help](https://help.emu.axiell.com/v5.1/en/Resources/Downloads/Unicode/EMu_Unicode_Cheatsheet_IE_20171019.pdf))
- [Keyboard Shortcuts Quick-ref](https://github.com/fieldmuseum/EMu-Documentation/files/14015690/EMuV5_and_earlier_QuickRef_Guide_IE_20170630.pdf) - (from [Axiell help](https://help.emu.axiell.com/v5.1/en/Resources/Downloads/Quick%20Reference%20Guide/EMuV5_and_earlier_QuickRef_Guide_IE_20170630.pdf))
 
Host = 10.10.10.25  **OR** shackleton.fm.pri
(Either the number or shackleton.fm.pri not both)
 
Troubleshooting/Questions:
- [Errors with opening pdf's](https://github.com/fieldmuseum/EMu-Documentation/discussions/366)
- [What about Mac's?](https://github.com/fieldmuseum/EMu-Documentation/discussions/419)
- Contact the helpdesk x4357 or [helpdesk@fieldmuseum.org](mailto:helpdesk@fieldmuseum.org)

**Reply by rondlg** [reply]
@magpiedin @fmjjones @pdavisfmnh 

Post the move off of filer01 and the upgrade to v9 this discussion will need to be updated with new user instructions.

**Comment by rondlg** [comment]
## v9 instructions - FM Managed machines

*****BEFORE BEGINNING THE PROCESS, READ THE INSTRUCTIONS*****

The emu client is distributed by the Museums management platform.

On your computer launch the self service portal. 
- There should be a shortcut on your desktop with the blue/white square-shaped Field Museum logo.
- If you don't see the shortcut, click the 'Windows' icon, and type: "Field Museum Endpoint Central Cloud"
- <img width="450" alt="Windows Start Menu search" src="https://github.com/user-attachments/assets/b27cc3ec-2662-4bc2-a947-8b579777dccf" />


In the list of software available to install you will find "KE Emu (V9)"
- If you don't see it, try clicking the circular 'Refresh' icon in the upper-right of the portal window.
- <img width="475" alt="the FM self service portal" src="https://github.com/user-attachments/assets/6a15dfc2-ba3f-4966-bca5-19466a94eaa8" />


Click on the Install button

**Comment by pdavisfmnh** [comment]
## v9 instructions - NON-FM-Managed machines

1. Get the latest EMu LIVE install-files

- Open a file browser and go to \\filer03.fm.pri\emu\clients\live\
- Copy the files to your computer

2. Install Emu

- Double click on the install_client.bat file. 
- This will install the client with all of the proper settings silently in the background. 
- The EMu icon will now show up on your desktop.

3. Log into the LIVE environment
